metadata.json

```
@@ -10,7 +10,7 @@
   "dependencies": [
     {
       "name": "puppetlabs/stdlib",
-      "version_requirement": ">= 4.2.0 < 9.0.0"
+      "version_requirement": ">= 4.2.0 < 10.0.0"
```

This should raise the lower bound since it's using the namespaced version.
```suggestion
      "version_requirement": ">= 9.0.0 < 10.0.0"
```